### DISCUSSION TITLE
Open source for all: Skilling the next generation of developers

### DISCUSSION THREAD

**Discussion by Ruth-ikegah** [community/community#141485] [Other Feature Feedback, Questions, & Ideas]
Universe 2024 Discussion Lounge Session: [Open source for all: Skilling the next generation of developers](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715564726257001Azx9)

> There are similarities between diversity and inclusion efforts in tech today and software development in the early days of open source. That is to say: even with many resources and funds invested, efforts are often disjointed, siloed, or sidelined. As GitHub scales to becomes home for one billion global developers, we aim to fix this. Join facilitators, Ruth Ikegah, open source consultant at CHAOSS, and Sarah Oyetubo, senior DI&B program manager at GitHub, for a discussion on how to foster collaboration and help skill new developers, lowering the barrier of entry to open source for all.

We would love to hear from you, how are you skilling developers about the projects you build in your communities, and what challenges have you faced with this?

1. How can we use current open source projects to set up mentorship programs that help developers in the Global South improve their skills and build a supportive learning community? ​Share examples of successful approaches you've seen or have been a part of.​
2. What effective strategies have you seen that improve access to open source training resources for aspiring developers in the Global South, especially those that make these resources more relevant and easier to use?​
3. As we witness a surge in tech and open source job opportunities across the Global South, how can we create pathways for contributors from underrepresented groups? ​What specific actions, resources, or support systems do you think would be effective in ensuring that these individuals are included in economic growth?​

Drop your comments in the thread 👇👇

**Comment by Blockcats** [comment]
توضیح آینده خوبه قاطع میگی از لذت یک راه برتر و بی رقیب .و پیشرو.

**Reply by ghostinhershell** [reply]
Hi @Blockcats, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. Thank you!

**Reply by dawidrybarczyk43** [reply]
> توضیح آینده خوبه قاطع میگی از لذت یک راه برتر و بی رقیب .و پیشرو.

 PayPal bank dawidrybarczyk43@gmail.com bimber +48662146385

**Comment by Gesronet** [comment]
Thank you for the great questions!

**Skilling Developers:** We focus on hands-on workshops and collaborative projects, though varying skill levels can be a challenge.

**Mentorship Programs:** Using open-source projects for mentorship is effective. Pairing experienced developers with newcomers on real projects builds skills and confidence.

**Access to Training Resources:** Curating localized content and creating interactive platforms for questions can improve access to training.

**Pathways for Underrepresented Groups:** Intentional outreach, scholarships, and partnerships with local organizations can support diversity and inclusion in tech.

Looking forward to more ideas from the community!

**Comment by Blockcats** [comment]
زبان فارسی در گفتار خروجی یکی از مشکلات من است امکان دسترسی به صدا فارسی در
آفلاین وجود دارد پاسخگو باشید ممنون

در تاریخ دوشنبه 18 نوامبر 2024،‏ 7:19 Samuel Otomewo <
***@***.***> نوشت:

**Comment by Blockcats** [comment]
Language for Persian Speak app offline reder. Gtts installer chang mi
persian?

در تاریخ چهارشنبه 30 اکتبر 2024،‏ 23:52 Sarto ***@***.***>
نوشت:

**Comment by Blockcats** [comment]
I have to be sorry, I know English in a broken way and I don't have Persian
voice offline, it is possible to check that I can do the text by voice and
have a comprehensive language change.

در تاریخ دوشنبه 18 نوامبر 2024،‏ 8:20 Ars Lan ***@***.***> نوشت:
